### PR TITLE
fix: [SRM-16344]: Update SLO DTOs.

### PR DESCRIPTION
**Pull request by naveen-harness**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- gitleaks: `trigger gitleaks`

### CODE REVIEW

File: harness/nextgen/model_service_level_indicator_dto.go [hunk 1]
```diff
@@ -9,11 +9,17 @@
...
+	Name       string                                 `json:"name,omitempty"`
+	Identifier string                                 `json:"identifier,omitempty"`
+	Type_      SLIEvaluationType                      `json:"type,omitempty"`
+	Window     *WindowBasedServiceLevelIndicatorSpec  `json:"-"`
+	Request    *RequestBasedServiceLevelIndicatorSpec `json:"-"`
+	MetricLess *MetricLessServiceLevelIndicatorSpec   `json:"-"`
+
+	Spec               json.RawMessage `json:"spec"`
+	SliMissingDataType string          `json:"sliMissingDataType"`
+	HealthSourceRef    string          `json:"healthSourceRef,omitempty"`
```
Comment (deepak-harness) [inline harness/nextgen/model_service_level_indicator_dto.go:24]: you can remove this. since this field is depricated.

Comment (naveen-harness) [reply harness/nextgen/model_service_level_indicator_dto.go]: Done

**Comment by naveen-harness** [comment]
trigger build

**Review by deepak-harness — APPROVED** [review]

**Comment by naveen-harness** [comment]
trigger build

**Pull request merged**